dev.sh

```diff
@@ -17,13 +17,13 @@ export BRAINLIFE_AUTHENTICATION
 
 git submodule update --init --recursive
 
-(cd api && npm install -g npm@9.5.1)
-(cd ui && npm install -g npm@9.5.1)
+(cd api && npm install)
+(cd ui && npm install)
```

Comment:
@dlevitas @anibalsolon  Mac is such a nightmare, I had to mess with this stuff and disable husky to get this running on my x86 machine (old intel mac pro). As I mention in the PR description this _should_ probably be undone, will do so once I've got https working properly.